modules/openapi-generator/src/test/resources/3_0/petstore.yaml

```diff
@@ -94,6 +94,14 @@ paths:
                 - pending
                 - sold
               default: available
+        - name: booltest
+          in: query
+          description: test for boolean parse generation
+          required: true
+          schema:
+            type: boolean
+            default: true
+            example: true
```

Comment:
Please do not change petstore.yaml.

To have more test cases covered, please switch to https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing.yaml instead.